src/webextension/list/popup/components/app.css

```diff
@@ -14,5 +14,5 @@ body {
 
 .app {
   width: 280px;
-  height: 400px;
+  height: 360px;
```

Comment:
Can we also move this up to the `body` (`width` and `height`) like in the unlock view? Then we can probably remove this selector entirely (and its reference in `app.js`).